src/js/components/agencyLanding/table/AgencyLandingTable.jsx

```diff
@@ -0,0 +1,158 @@
+        // determine the table position
+        let yPosition = yPos;
+        if (!returnTop) {
+            // return the bottom row
+            yPosition += tableHeight;
+        }
+        return Math.floor(yPosition / rowHeight);
+    }
+
+    rowClassName(index) {
+        let evenOdd = 'odd';
+        if ((index + 1) % 2 === 0) {
+            evenOdd = 'even';
+        }
+        return `agency-landing-results-row-${evenOdd}`;
+    }
+
+    prepareTable() {
+        let totalWidth = 0;
+
+        const HeaderCell = this.props.headerCellClass;
+
+        const columns = this.props.columns.map((column, i) => {
+            // For this table, make each column's width a percentage of the visible width
+            const adjustedWidth = (this.props.visibleWidth * column.width);
+            totalWidth += adjustedWidth;
+            const isLast = i === this.props.columns.length - 1;
+            let cellName = null;
+            if (column.columnName === 'agency_name') {
+                cellName = (index) => (
+                    <AgencyLinkCell
+                        key={`cell-${column.columnName}-${index}`}
+                        rowIndex={index}
+                        data={this.props.results[index][column.columnName]}
+                        dataHash={this.state.dataHash}
+                        column={column.columnName}
+                        isLastColumn={isLast} />
+                );
+            }
+            else {
+                cellName = (index) => (
+                    <ResultsTableGenericCell
+                        key={`cell-${column.columnName}-${index}`}
+                        rowIndex={index}
+                        data={this.props.results[index][column.columnName]}
+                        dataHash={this.state.dataHash}
+                        column={column.columnName}
+                        isLastColumn={isLast} />
+                );
+            }
+            return {
+                width: adjustedWidth,
+                name: column.columnName,
+                columnId: `${column.columnName}`,
+                rowClassName: this.rowClassName,
+                header: (
+                    <HeaderCell
+                        label={column.displayName}
+                        column={column.columnName}
+                        defaultDirection={column.defaultDirection}
+                        isLastColumn={isLast} />
+                ),
+                cell: cellName
+            };
+        });
+
+        return {
+            columns,
+            width: totalWidth
+        };
+    }
+
+    render() {
+        const calculatedValues = this.prepareTable();
+
+        let noResultsClass = '';
+        if (this.props.results.length === 0) {
+            // remove duplicated bottom border
+            noResultsClass = ' no-results';
+        }
+
+        return (
+            <div className={`agency-landing-results-table${noResultsClass}`}>
+                <IBTable
```

Comment:
I'm not sure that IBTable is the right use case for this table. It uses fixed column widths and forces scrolling within the set view. A standard table may be easier.